src/translate_rapid_pro_keys.py

```diff
@@ -108,6 +108,12 @@ def remap_radio_shows(cls, user, data, pipeline_configuration):
                 user, data, remapping.time_key, remapping.show_pipeline_key_to_remap_to,
                 remapping.range_start_inclusive, remapping.range_end_exclusive, remapping.time_to_adjust_to
             )
+            # Correct diagnostic s01e03 messages recovered from the All networks downtime issue
+            cls._remap_radio_show_by_time_range(
```

Comment:
You can set this from configuration json

Reply:
moved to pipeline configuration json file.